scipy/sparse/linalg/_svdp.py

```diff
@@ -79,10 +79,11 @@ def dtype(self):
             return self.A.matvec(np.zeros(self.A.shape[1])).dtype
 
 
+@_transition_to_rng("random_state", position_num=18)
```

Comment:
Does this need the decorator to be applied? It's private, so it's OK to just change `random_state` to `rng` within.

Reply:
I wasn't sure how "private" it is because it is listed in `__all__`.  But it looks like no `__init__.py` files import it like that. So, in SciPy, it is only used by `svds` and in the associated tests for `propack`. I removed it here. A search for `"_svdp" site::https://github.com` shows only 5 results non of which are relevant. It's safe to say it isn't use elsewhere. I've updated accordingly here and in `_svds.py`.